fedimint-server/src/consensus/api.rs

```diff
@@ -726,5 +759,13 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
                 Ok(fedimint.fedimintd_version())
             }
         },
+        api_endpoint! {
+            BACKUP_STATISTICS_ENDPOINT,
+            ApiVersion::new(0, 5),
+            async |fedimint: &ConsensusApi, context, _v: ()| -> BackupStatistics {
+                check_auth(context)?;
+                Ok(fedimint.backup_statistics(&mut context.dbtx().into_nc()).await)
```

Comment:
This seems relatively heavy IO-wise, so maybe we should cache it for at least a minute.

Reply:
Oh, we require Auth. Nvmd then.